public/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Freedomvote</title>
+    <link rel="stylesheet" href="main.css">
+  </head>
+  <body>
+    <header class="navigation">
+      <nav class="navigation__nav">
+        <a href="#">
+          <img class="navigation__nav__logo" src="/img/logo.svg" />
+        </a>
+        <ul class="navigation__nav__page">
+          <li><a href="#" class="navigation__nav__page__active">Home</a></li>
+          <li><a href="#">Kandidaten</a></li>
+          <li><a href="#">Vergleichen</a></li>
+          <li><a href="#">Partner</a></li>
+        </ul>
+        <i class="navigation__nav__divider"></i>
+        <nav class="navigation__nav__language">
```

Comment:
Create a own component for this element. So that you can take it and place anywhere else.